com_dpfields/admin/helpers/dpfields.php

```diff
@@ -187,6 +197,14 @@ public static function getFields ($context, $item = null, $prepareValue = false)
 						));
 					}
 
+					/*
+					 * On after field render
+					 * Event allow plugins to modfify the output of the prepared field
+					 */
+					if ($value)
```

Comment:
That is true but at the moment the before event has the same check as the after event and so it makes no difference.